server/routerlicious/packages/protocol-base/src/quorum.ts

```diff
@@ -92,6 +92,7 @@ export class QuorumClients extends TypedEventEmitter<IQuorumClientsEvents> imple
      * Adds a new client to the quorum
      */
     public addMember(clientId: string, details: ISequencedClient) {
+        assert(!!clientId, "clientId has to be non-empty string");
```

Comment:
Should the assert be some unique number like the line below it?

Reply:
It will become it - after I merge it and we cut release - it's a part of postprocessing / release flow.